hermeznetwork/hermez-node
# Feature/#711 default config

Closes #711

### What does this PR does?

This PR adds the functionality to support configuring the hermez node via env variables. If this env variables are not setted, then it uses the variables inside the config file. If there are some config params that are still missing, the hermez node will use the default ones.
Also It uses the config parser as library

Batches forged:
https://explorer.internaltestnet.hermez.io/batch/27288  using env variables
https://explorer.internaltestnet.hermez.io/batch/27301 using config file
https://explorer.internaltestnet.hermez.io/batch/29058  without setting ethscan parameters
#####################
https://explorer.internaltestnet.hermez.io/batch/29177  using env variables without config file
https://explorer.internaltestnet.hermez.io/batch/29180 without etherscan
https://explorer.internaltestnet.hermez.io/batch/29183  with env and config file
https://explorer.internaltestnet.hermez.io/batch/29184  using wrong config file name and env
https://explorer.internaltestnet.hermez.io/batch/29186  using only config file

### How to test?

Run hermez node setting env variables.
Run hermez node without env variables or toml file to check that it is getting the default variables correctly.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
